css-layout-classes/index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Layout Classes</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <header>
+            <h1>Choose your character!</h1>
+          </header>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-one-third">
+          <article>
+            <img class="character" src="/images/link.png" alt="link" />
+          </article>
+        </div>
+
+        <div class="column-one-third">
+          <article>
+            <h2>Super Smash Bros. Ultimate</h2>
+            <p>
+              <strong>
+                Super Smash Bros. Ultimate received universal acclaim from both
+                critics and players, with some critics calling it the best
+                installment in the series. It received praise for its large
+                amount of content and fine-tuning of existing Smash gameplay
+                elements, although its online mode was widely
+                criticized.</strong
+              >
+            </p>
+          </article>
+        </div>
+        <div class="column-one-third">
+          <aside>
+            <img class="character" src="/images/yoshi.png" alt="yoshi" />
+          </aside>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-one-fourth">
+          <div>
+            <img class="character" src="images/kirby.png" alt="kirby" />
+          </div>
+        </div>
+        <div class="column-one-fourth">
+          <div>
+            <img class="character" src="images/fox.png" alt="fox" />
+          </div>
+        </div>
+        <div class="column-one-fourth">
+          <div>
+            <img class="character" src="images/pikachu.png" alt="pikachu" />
+          </div>
+        </div>
+        <div class="column-one-fourth">
+          <div>
+            <img class="character" src="images/ness.png" alt="ness" />
+          </div>
+        </div>
+      </div>
```

Comment:
Too many `div`s. Let your `img` be a direct child of your `div.column-one-fourth`